test/index.test.js

```diff
@@ -0,0 +1,44 @@
+const MemoryDatabase = require('../lib/index');
```

Comment:
```suggestion
const MemoryDatabase = require('../lib/index');

```
Normally, there would be a line to divide the imports from the code blocks to improve readability.